resources/js/utils/diffForHumans.ts

```diff
@@ -5,10 +5,8 @@ export function diffForHumans(date: string, from?: string): string {
   const end = dayjs(date);
   const diff = Math.abs(start.diff(end, 'second'));
 
-  if (diff < 1) {
-    return 'now';
-  } else if (diff < 60) {
-    return `${Math.floor(diff)} second${diff === 1 ? '' : 's'} ${start.isAfter(end) ? 'ago' : 'from now'}`;
+  if (diff < 60) {
+    return 'A few seconds ago';
```

Comment:
Doesn't this just shift the window such that something at diff==59 would send "A few seconds ago", but the hydration would see "1 minute ago"?

Also 59 is not "a few". Maybe we should have "less than 1 minute" for 6-59 seconds?

Reply:
Yep, you're right. I've landed on a more robust solution in latest via a custom hook:
```ts
export function useServerRenderTime() {
  // `useState` runs on the server, so this value is populated during SSR.
  const [renderedAt, setRenderedAt] = useState(() => dayjs.utc().toISOString());

  // `useEffect` runs only on the client. The `renderedAt` state will try to
  // "drift" during hydration. Have the effect pin the value.
  useEffect(() => {
    setRenderedAt(renderedAt);
  }, [renderedAt]);

  return { renderedAt };
}
```

For all intents and purposes, this forces the value to never drift during hydration.

Additionally, I've adjusted the windows. <10s now reads "Just now", and 10-59s reads as "Less than a minute ago". Let me know if you think these windows or window labels should be further tweaked.